sphinx_tabs/tabs.py

```diff
@@ -25,7 +26,11 @@
 LEXER_MAP = {}
 for lexer in get_all_lexers():
     for short_name in lexer[1]:
-        LEXER_MAP[short_name] = lexer[0]
+        if short_name == "r":
+            # S by default
+            LEXER_MAP[short_name] = "R"
```

Comment:
Will drop this, as changes below allow you to specify the `code-tab` label to override the pygments names